pkg/testutil/dynamodb.go

```diff
@@ -53,19 +54,8 @@ func RunLocalDynamoDBInstance(port string) (string, func(), error) {
 
 	uri := fmt.Sprintf("http://localhost:%s", port)
 
-	// dummy aws env, in case not configured
-	if os.Getenv("AWS_DEFAULT_REGION") == "" {
-		os.Setenv("AWS_DEFAULT_REGION", "us-west-1")
-	}
-	if os.Getenv("AWS_ACCESS_KEY_ID") == "" {
-		os.Setenv("AWS_ACCESS_KEY_ID", "fakeMyKeyId")
-	}
-	if os.Getenv("AWS_SECRET_ACCESS_KEY") == "" {
-		os.Setenv("AWS_SECRET_ACCESS_KEY", "fakeSecretAccessKey")
-	}
-
 	// Disgusting, but I have to wait for the dynamodb container to be ready
-	time.Sleep(2 * time.Second)
+	time.Sleep(WaitForContainerSec * time.Second)
```

Comment:
Let's probe the container healthcheck instead. Not only this is "Disgusting", it is also flaky and does not guarantee that the container is up and healthy.

Reply:
Done